ci-operator/step-registry/ingress/aws/conf/subnets/ingress-aws-conf-subnets-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: ingress-aws-conf-subnets
+  from_image:
+    namespace: ocp
+    name: cli-yq
```

Comment:
Seem like you did not use `yq` command in this step, so I suggest to use `from: cli`